src/patternfly/components/Badge/styles.scss

```diff
@@ -0,0 +1,46 @@
+@import "../../patternfly-utilities";
+
+:root {
+  --pf-c-badge--BorderRadius: var(--pf-global--BorderRadius);
+  --pf-c-badge--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-badge--FontWeight: var(--pf-global--FontWeight--bold);
+  --pf-c-badge--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-badge--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-badge--m-read--BackgroundColor: var(--pf-global--BackgroundColor--light-300);
+  --pf-c-badge--m-read--Color: var(--pf-global--dark--Color);
+  --pf-c-badge--m-read-alt--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-badge--m-read-alt--Color: var(--pf-global--Color--dark-200);
+  --pf-c-badge--m-unread--BackgroundColor: var(--pf-global--primary-color--dark);
+  --pf-c-badge--m-unread--Color: var(--pf-global--Color--light-100);
+  --pf-c-badge--m-unread-alt--BackgroundColor: var(--pf-global--BackgroundColor--light-100);
+  --pf-c-badge--m-unread-alt--Color: var(--pf-global--primary-color);
+}
+
+.pf-c-badge {
+  display: inline-block;
+  padding-right: var(--pf-c-badge--PaddingRight);
+  padding-left: var(--pf-c-badge--PaddingLeft);
+  font-size: var(--pf-c-badge--FontSize);
+  font-weight: var(--pf-c-badge--FontWeight);
+  border-radius: var(--pf-c-badge--BorderRadius);
+
+  &.pf-m-read {
+    color: var(--pf-c-badge--m-read--Color, #292e34);
+    background-color: var(--pf-c-badge--m-read--BackgroundColor);
+
+    &-alt {
+      color: var(--pf-c-badge--m-read-alt--Color);
+      background-color: var(--pf-c-badge--m-unread-alt--BackgroundColor);
+    }
+  }
+
+  &.pf-m-unread {
```

Comment:
I'm not sure about read, unread for these. Is it too specific? Naming is hard :)

Reply:
I chatted with @mceledonia about this yesterday. He recommended calling them "read" and "unread" because

> I think it might be best to keep them pretty rigid in use, so that they’re always used in the same fashion

Reply:
what are your thoughts on it? Does blue always mean unread? Is it possible to use a badge but not want to highlight it that much?

Reply:
I'm not familiar enough with using badges to have a strong opinion one way or the other. I like the idea of leaving them flexible for use as folks see fit, but I can also get behind naming them specifically to encourage devs to use them consistently for their intended purpose.

Reply:
I thought one of the goals of patternfly-next was to provide components that are modular and a design system that's flexible enough to accommodate any possible design that a product needs, versus designs that are specific and support a narrow set of use cases. If that is the case, then I'd suggest using labels that are generic because to me, this component also seems like a very general purpose one. But I'm also not familiar with the design documentation associated with it enough to know if this is a general purpose component or one with a very specific purpose.

Reply:
@jgiardino ya, this might be something @mcarrano needs to chime in on.

Reply:
@jgiardino @matthewcarleton Agreed it would be good to get some interaction input on this. The reason behind my initial decision was that this to me was similar to something like input field states, there's a default, then other states based on conditions brought by the user's actions (active, disabled, etc.). Similarly I designed the badges to have multiple "states" as you commonly see, one for "there are this many items" and a second one for "there are this many items, and also new stuff you haven't seen yet." The reason I figured it would be good to go with specific naming is because we don't want people reversing those patterns (especially internally) and cause inconsistency/confusion in products.

From an external, open-source perspective (to Jenn's point), I was assuming that these 2 states remain the same, "This many things" and "This many things with some new stuff." I am not sure of any other use cases for numeric badges like this but I could easily be overlooking those. 

So it sounds like the balance here is making sure we can have consistent usage between the states but also not limiting use cases that might extend outside our original intentions behind the badges. I don't feel strongly one way or the other, and it's a good point "read" and "unread" might be too specific. Users might not even want to have both states for whatever reason, they might just want to display a simple numeric count. 

The way I've been describing them here is too long (this many things) but it gives a sense of how I envisioned the scope of the use cases for the two states.